packages/rich-text/src/plugins/CommandPalette/components/CommandList.tsx

```diff
@@ -0,0 +1,163 @@
+import * as React from 'react';
+
+import { Popover, Stack, SectionHeading, ScreenReaderOnly, Flex } from '@contentful/f36-components';
+import { PlateEditor } from '@udecode/plate-core';
+import { cx } from 'emotion';
+
+import { useSdkContext } from '../../../SdkProvider';
+import { useCommandList } from '../hooks/useCommandList';
+import { CommandList as CommandItems, useCommands } from '../useCommands';
+import styles from './CommandList.styles';
+
+export interface CommandListProps {
+  query: string;
+  editor: PlateEditor;
+}
+
+const CommandListItems = ({
+  commandItems,
+  selectedItem,
+}: {
+  commandItems: CommandItems;
+  selectedItem: string;
+}) => {
+  const group = (command) => {
```

Comment:
`group`, `asset` and `item` are all React components essentially, declaring them inside the body of `CommandListItems` is inefficient as the references are being recreated in every `render`.

What's more, they should use Pascal case. I'd be happy to keep them in the same file, but I'd propose moving them outside of `CommandListItems` body.

Reply:
🤦‍♂️ I don't know where my head was yesterday
Thanks for spotting it!